### PR TITLE
Include the identifier in the call to FBSnapshotVerifyView for SnapshotVerifyWithInvertedColors

### PR DESCRIPTION
**Pull request by kruegermj**
Resolves https://github.com/cashapp/AccessibilitySnapshot/issues/51

### CODE REVIEW

File: Example/AccessibilitySnapshot/InvertColorsViewController.swift [hunk 1]
```diff
@@ -33,8 +33,8 @@ final class InvertColorsViewController: AccessibilityViewController {
         view = View()
     }
 
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+    override func viewDidLoad() {
+        super.viewDidLoad()
```
Comment (NickEntin) [inline Example/AccessibilitySnapshot/InvertColorsViewController.swift:37]: Why the change to `viewDidLoad()`? This should mirror the notification center observer removal in `viewWillDisappear(_:)` below.

Comment (kruegermj) [reply Example/AccessibilitySnapshot/InvertColorsViewController.swift:37]: I misunderstood the comment below in the [issue](https://github.com/cashapp/AccessibilitySnapshot/issues/51). I think we just need to update the status label in viewDidLoad. I made that change. 

> Due to a view life cycle issue with the snapshot test framework, you'll need to modify where the status get checked in the sample view controller in order to get the initial check to pass. The easiest way to accomplish this is to move the code from viewWillAppear(_:) to the end of viewDidLoad().

**Review by NickEntin — APPROVED** [review]
